runner/docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3.8'
+
+services:
+  pyro-engine:
+    build:
+      context: ..
+      dockerfile: ./runner/src/Dockerfile
+    command: python runner.py
+    volumes:
+      - ./data:/usr/src/app/data
+
+    restart: always
```

Comment:
Considering the simplicity of this, we might consider in the future to run it as a simple docker (not a docker compose)

Reply:
We can leave it as it is for know and discuss it soon :)